.github/workflows/generation.yml

```diff
@@ -34,53 +41,50 @@ jobs:
             --additional-properties=modelPackage=models
             --additional-properties=apiPackage=api
             --enable-post-process-file
-            -o /github/workspace/unstable
-
-      - name: Get server commit hash
-        id: sha
-        run: |
-          SERVER_SHA="$(git ls-remote git://github.com/jellyfin/jellyfin.git | grep refs/heads/master | cut -f 1)"
-          echo "::set-output name=sha::$(echo ${SERVER_SHA::7})"
+            -o /github/workspace/${{ matrix.version }}
 
       - name: Check if .ts files were modified
         id: diff
-        run: echo "::set-output name=count::$(git status -s | grep ts | wc -l)"
+        run: echo "::set-output name=count::$(git status -su | grep ts | wc -l)"
 
-      - name: Get OpenApi version
+      - name: Set docker files ownership to runner
         if: ${{ steps.diff.outputs.count > 0 }}
-        id: apiversion
-        run: |
-          SERVER_VERSION="$(cat openapi.json | jq '.info.version')"
-          echo "::set-output name=number::$(echo ${SERVER_VERSION::-1})"
+        run: sudo chown -R runner:docker .
```

Comment:
Why?

Reply:
It may not be, it's just that any file edited by the docker action will be owned by root. So if package.json or package-lock.json are edited during the generation, we can't edit them afterwards with the version.

Are they potentially edited during the generation?

Reply:
I agree, this is not needed :? It has worked up to this point without permission issues so I would argue this is redundant and should only be fixed if we ever notice it causing a problem